app/models/bundle_context.rb

```diff
@@ -95,8 +95,8 @@ def manifest_rows
 
   def manifest_cols
     {
-      label: 'label',
-      source_id: 'sourceid',
+      label: 'label', # TODO: no longer needed (#285)
+      source_id: 'sourceid', # TODO: no longer needed (#285)
```

Comment:
actually, it's possible this comment is incorrect for smpl manifests ... I need to check.